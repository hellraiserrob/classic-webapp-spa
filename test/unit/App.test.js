import { shallow } from '@vue/test-utils'
import App from '../../src/scripts/components/App.vue';

describe('App.vue', () => {
  
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(App);
  })

  it('renders expected html', () => {
    const btn = wrapper.find('button');
    expect(btn.text()).toMatch('toggle...');
  });

  it('Should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });

})
