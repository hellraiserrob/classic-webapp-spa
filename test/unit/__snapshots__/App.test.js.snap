// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue Should match snapshot 1`] = `
<div>
    <button>toggle...</button>
    <!---->
</div>
`;
